echo-webhooks/src/main/groovy/com/netflix/spinnaker/echo/controllers/WebhooksController.groovy

```diff
@@ -73,6 +75,7 @@ class WebhooksController {
     if (headers.containsKey('X-Event-Key')) {
       event.content.event_type = headers['X-Event-Key'][0]
     }
+    def filteredHeaders = CollectionUtils.toMultiValueMap(headers.findAll { it.key.startsWith("x-github") })
```

Comment:
Does casing matter on the header? The headers above are cased, and [this doc](https://docs.github.com/en/rest/guides/getting-started-with-the-rest-api#hello-world) has a mixture.

Reply:
When I tested it I received lowercased headers. But good point, I can make the check ignore casing.